mne_features/bivariate.py

```diff
@@ -103,6 +95,34 @@ def compute_max_cross_correlation(s_freq, data):
     return max_cc
 
 
+def compute_max_cross_correlation(sfreq, data):
```

Comment:
No need to add this to api.rst ?

Reply:
It's already in api.rst. 
Sphinx does not like jitted functions so I created a private version of the function so that the public version is a pure Python function.